src/main/java/io/vertx/core/http/impl/Http1xClientConnection.java

```diff
@@ -69,6 +70,9 @@
   private static final Logger log = LoggerFactory.getLogger(Http1xClientConnection.class);
 
   private static final Handler<Object> INVALID_MSG_HANDLER = msg -> {
+    if (msg instanceof ReferenceCounted) {
```

Comment:
I believe there should be an Util class from Netty to do the same

Reply:
ok, if you know which one :-)

Reply:
`ReferenceCountUtil::release`!

Reply:
I'll use it asap then to cleanup this

Reply:
https://github.com/eclipse-vertx/vert.x